tools/testbuild.sh

```diff
@@ -258,51 +258,55 @@ function build {
 function dotest {
   echo "===================================================================================="
   config=`echo $1 | cut -d',' -f1`
-  re="-${config/\//:}[[:space:]]"
-  if [[ "${blacklist} " =~ $re ]]; then
-    echo "Skipping: $1"
-  else
-    echo "Configuration/Tool: $1"
-    if [ ${PRINTLISTONLY} -eq 1 ]; then
+  check=-`uname -s`,${config/\//:}
```

Comment:
When I was setting up the windows CI I did this to help ID the system. Maybe we should do the same here since uname is not always exact especially between 32/64bit. Then we also have and explicit list.

```
kernel="$(uname -s)"
case $kernel in
 Linux*) os=Linux;;
 Darwin*) os=Darwin;;
 CYGWIN*) os=Cygwin;;
 MINGW*) os=MinGw;;
 *) os="UNKNOWN:$kernel"
esac
```

Reply:
Done.